### PR TITLE
Add support for computed properties

### LINKED ISSUE #5: Computed properties
Add support for computed properties to reflect [`classnames`](https://www.npmjs.com/package/classnames) API.

String literals must act like regular property names:
```js
const CLASS_NAMES = classNames({
  ["foo"]: props.foo,
});
// => "" + (props.foo ? " foo" : "");
```

Identifiers must be unwrapped into conditional expressions:
```js
const CLASS_NAMES = classNames({
  [bar]: props.bar,
});
// => "" + (props.bar ? bar ? " " + bar : "" : "");
```

Member expressions must be unwrapped into conditional expressions:
```js
const CLASS_NAMES = classNames({
  [styles.baz]: props.baz
});
// => "" + (props.baz ? styles.baz ? " " + styles.baz : "" : "");
```

### PR DESCRIPTION
**Pull request by kinday**
Resolves #5

### CODE REVIEW

**Comment by daviderenger** [comment]
🎉

**Comment by kinday** [comment]
Hey @daviderenger! Would you mind testing it out before I merge?

**Comment by daviderenger** [comment]
Yeah sure, I can test it later today, in about 5 hours or so

File: src/classnames.macro.js [hunk 1]
```diff
@@ -35,13 +35,26 @@ export default createMacro(({ babel, references: { default: paths } }) => {
     return babel.types.arrayExpression(elements)
   }
 
+  function unpackObjectProperty(objectProperty) {
+    console.log(objectProperty)
```
Comment (daviderenger) [inline src/classnames.macro.js]: Is this stripped in final build? It is extremely verbose when testing anyway :)

**Review by daviderenger — CHANGES_REQUESTED** [review]

**Comment by daviderenger** [comment]
Sorry for the delay, life came in between :grin: 

I have tested this and it works like a charm! Solves the problem for us anyway and doesn't break anything I can see. The only thing is the console.log mentioned above

File: src/classnames.macro.js (hunk 1 above)
Comment (kinday) [reply src/classnames.macro.js]: Uh no. It should not be there, development process leftover. Good catch.

**Pull request merged**